os/fs/driver/mtd/smart.c

```diff
@@ -1855,7 +1855,7 @@ static int smart_scan(FAR struct smart_struct_s *dev)
 
 		status_released = SECTOR_IS_RELEASED(header);
 		status_committed = SECTOR_IS_COMMITTED(header);
-#ifdef CONFIG_MTD_SMART_JOURNALING
+#if defined(CONFIG_MTD_SMART_JOURNALING) && defined(CONFIG_DEBUG_FS_INFO)
```

Comment:
That's `fdbg`, not `fvdbg so that you should cover that with `CONFIG_DEBUG_FS_ERROR`.